ffwd-reporter/src/main/java/com/spotify/metrics/ffwd/FastForwardReporter.java

```diff
@@ -238,18 +238,19 @@ private void report(
     private void reportGauge(
         MetricId key, @SuppressWarnings("rawtypes") Gauge value
     ) {
-        if (value == null) {
```

Comment:
As `value` can never be null here, I assume the null check was simply misplaced.